readme.md

```diff
@@ -124,5 +140,6 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
-
+So with our team, Joe created the git repo, added added me and Alea as collaborators, we both cloned the repot and started adding feature branches. Near the
+end of the project we ran into some merge conflicts that we found easier to solve by creating pull requests.
 ```
```

Comment:
Like that you incorporated some personal experience. There are multiple right answers here. Here are a few mentioned in the quiz solution...

```
There are plenty of workflows a team can use:

1. Everyone is a contributor, and simply pushes their work to master.
2. Everyone is a contributor, but pushes their work to feature branches. Other team members merge in the branches at their discretion. Pull Requests may or may not be used.
3. Everyone forks the repo, and creates Pull Requests from their fork to the master. One or more people are in charge of reviewing and approving Pull Requests.

In addition to the above, teams may set their own policies on code review, when PRs can be merged, etc.

Additionally, teams may decide on a convention for naming their branches, and how changes get merged into the master branch (e.g. there may be an intermediate Development branch for integrating multiple PRs and testing how they work together.) One such example is the git flow model.
```